newscoop/library/Newscoop/Services/TemplatesService.php

```diff
@@ -66,5 +130,13 @@ private function preconfigureSmarty()
 
         // reverse templates dir order
         $this->smarty->setTemplateDir(array_reverse($this->smarty->getTemplateDir()));
+
+        $publicationMetadata = $this->publicationService->getPublicationMetadata();
+        if (count($publicationMetadata) > 0) {
+            $this->smarty->campsiteVector = array(
```

Comment:
Shouldn't we also include a unique key to the Newscoop install in this vector?  I'm thinking about this talk we had about cache for the deployment system. Or is this something else?